README.md

```diff
@@ -59,7 +59,7 @@ Since OpenMLDB v0.3.0, we introduce two operating modes, which are cluster mode
 
 We demonstrate the workflow of using the cluster and standalone modes:
 
-- Cluster mode: please read the :point_right: [quickstart guide](docs/en/cluster.md) and :point_right: [related demo code](demo/predict-taxi-trip-duration-nb/demo)
+- Cluster mode: please read the :point_right: [quickstart guide](docs/en/cluster.md) and :point_right: [related demo code](demo/predict-taxi-trip-duration-nb/script)
```

Comment:
- I think here we may link to the demo doc here `demo/` rather than the code itself
- We may add a separate desc saying we have a demo for both standalone and cluster `[here](demo/)`

Reply:
resolve in next pr